README.md

```diff
@@ -1,88 +1,114 @@
-# Svelte Demo
+# Web Component Svelte Demo
 
 ![Travis (.org)](https://img.shields.io/travis/alaskaairlines/AuroSvelteDemo?style=for-the-badge) ![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/AlaskaAirlines/AuroSvelteDemo?style=for-the-badge)
 
-Interested in starting a new project using Auro and Svelte? This demo project is a complete example that includes basic setup. 
+An example Svelte project with Auro Components integrated. This app runs and works in all Alaska-supported browsers. Explore the project source to see the Auro Components being used in a frameworkless environment.
 
-To locally run this demo: 
-
-```bash
-$ git clone https://github.com/AlaskaAirlines/AuroSvelteDemo.git
-```
-
-The app runs and works in all [supported browsers](http://auro.alaskaair.com/support/browsersSupport). Please explore the project source to see how the Auro components are being used in a Svelte development environment. 
-
-This is a project template for [Svelte](https://svelte.dev) apps. It lives at [https://github.com/sveltejs/template](https://github.com/sveltejs/template).
-
-To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
+Below are instructions for adding Auro compatibility to your project. A familiarity with modern Javascript development and bundlers such as Webpack is expected.
```

Comment:
I wouldn't mention Webpack here. Maybe take out the second sentence entirely. That was a sentence from the vanilla JS demo which requires more wiring up bundlers and whatnot.

Reply:
Good catch. Removed.